pytest/tests/sanity/memtrie_disktrie_switch.py

```diff
@@ -244,38 +251,39 @@ def __deploy_contracts(self):
 
     def __create_accounts(self):
         """Creates the test accounts.
-        
+
         Waits for the create-account transactions to complete."""
+
+        # Update nonces for the validator accounts, as we will use them to create the new accounts.
+        self.__retrieve_nonces([node.signer_key for node in self.nodes])
+
+        create_account_tx_list = []
         account_keys = []
-        for account_id in ALL_ACCOUNTS:
+        for i in range(len(ALL_ACCOUNTS_PREFIXES)):
+            # Choose a validator node to sign the transaction for creating the new account.
+            # We do not use a single validator, since transactions and their nonces can be
+            # reordered, which will make the transactions with smaller nonces invalid.
+            signer_key = self.nodes[i % len(self.nodes)].signer_key
```

Comment:
If this code is used only for this test, in which we have 5 account and 5 nodes, can we just write `i` here?

Reply:
yeah i should be sufficient. done.